athena-aws-cmdb/athena-aws-cmdb.yaml

```diff
@@ -36,6 +36,9 @@ Parameters:
     Description: "WARNING: If set to 'true' encryption for spilled data is disabled."
     Default: 'false'
     Type: String
+  SpillSSEAlgorithm:
+    Description: 'SSE algorithm for S3 spills'
+    Type: String
```

Comment:
Since we are allowing only aes256, maybe we should only allow the value aes256 here?

Reply:
Defaults that we may remove later can cause problems later imo. I think it's fine as-is. In theory, the other option is `KMS`, which would require another parameter specifying which key to use. That would mean we remove the check here as well.

We also do this pattern of "anything other than `x` means false` in other parameters, so overall I'm ok with this.